### PR TITLE
Fix the janky audio output settings UI

### PR DESCRIPTION
**Pull request by zfox23**
Fixes [BUGZ-239](https://highfidelity.atlassian.net/browse/BUGZ-239).

### CODE REVIEW

File: interface/resources/qml/hifi/simplifiedUI/topBar/SimplifiedTopBar.qml [hunk 1]
```diff
@@ -203,7 +203,8 @@ Rectangle {
 
         Image {
             id: outputDeviceButton
-            property bool outputMuted: false
+            property bool outputMuted: AudioScriptingInterface.avatarGain === -60 &&
```
Comment (ZappoMan) [inline interface/resources/qml/hifi/simplifiedUI/topBar/SimplifiedTopBar.qml]: my heart just sunk

File: scripts/system/simplifiedUI/simplifiedUI.js [hunk 2]
```diff
@@ -238,40 +238,40 @@ function updateOutputDeviceMutedOverlay(isMuted) {
 }
 
 
-var savedAvatarGain = Audio.getAvatarGain();
-var savedInjectorGain = Audio.getInjectorGain();
-var savedLocalInjectorGain = Audio.getLocalInjectorGain();
-var savedSystemInjectorGain = Audio.getSystemInjectorGain();
+var savedAvatarGain = Audio.avatarGain;
+var savedInjectorGain = Audio.serverInjectorGain;
```
Comment (MiladNazeri) [inline scripts/system/simplifiedUI/simplifiedUI.js]: should this be savedServerInjectorGain to match the others?

File: interface/resources/qml/hifi/simplifiedUI/topBar/SimplifiedTopBar.qml (hunk 1 above)
Comment (zfox23) [reply interface/resources/qml/hifi/simplifiedUI/topBar/SimplifiedTopBar.qml]: why?

Comment (ZappoMan) [reply interface/resources/qml/hifi/simplifiedUI/topBar/SimplifiedTopBar.qml]: magic numbers

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19596/HighFidelity-Beta-PR15613-7a88c83.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19596/HighFidelity-Beta-Interface-PR15613-7a88c83.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19596/HighFidelity-Beta-PR15613-7a88c83.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19596/HighFidelity-Beta-Interface-PR15613-7a88c83.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19596/HighFidelity-Beta-PR15613-7a88c83.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19596/HighFidelity-Quest-Beta-PR15613-7a88c83.apk)

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19602/HighFidelity-Beta-PR15613-46ee148.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19602/HighFidelity-Quest-Beta-PR15613-46ee148.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19602/HighFidelity-Beta-PR15613-46ee148.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19602/HighFidelity-Beta-Interface-PR15613-46ee148.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19602/HighFidelity-Beta-PR15613-46ee148.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19602/HighFidelity-Beta-Interface-PR15613-46ee148.exe

**Pull request merged**